examples/megacli.cpp

```diff
@@ -3907,6 +3942,26 @@ void exec_sync(autocomplete::ACState& s)
         }
     }
 }
+
+void exec_syncconfig(autocomplete::ACState& s)
+{
+    if (s.words.size() == 1)
+    {
+        cout << "Current sync config: " << syncConfig.syncType <<
```

Comment:
We could use the new method `static std::string syncConfigToString(const Sync& sync)` in here?